bzlib.rs

```diff
@@ -10,9 +10,12 @@ use crate::decompress::{self, decompress};
 use crate::libbzip2_rs_sys_version;
 use crate::BZ_MAX_UNUSED;
 
+// FIXME remove this
 extern "C" {
-    static stdin: *mut FILE;
-    static stdout: *mut FILE;
+    #[cfg_attr(target_os = "macos", link_name = "__stdinp")]
+    static mut stdin: *mut FILE;
+    #[cfg_attr(target_os = "macos", link_name = "__stdoutp")]
+    static mut stdout: *mut FILE;
 }
```

Comment:
yeah these are also a problem on windows, leading me to suspect that this binary was not actually cross-platform . Invalid windows paths are handled, but this would not even build on windows...

Reply:
In C `stdin` and `stdout` are macros which expand to a libc specific expression. Unfortunately the libc crate doesn't provide the definitions of these macros, so we have to manually write their expansion instead.